sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2789,6 +2789,12 @@ class Dataset[T] private[sql](
    * When no explicit sort order is specified, "ascending nulls first" is assumed.
    * Note, the rows are not sorted in each partition of the resulting Dataset.
    *
+   *
+   * Note that due to performance reasons this method uses sampling to estimate the ranges.
+   * Hence, the output may not be consistent, since sampling can return different values.
+   * The sample size can be controlled by setting the value of the parameter
+   * `spark.sql.execution.rangeExchange.sampleSizePerPartition`.
```

Comment:
It's not a parameter but a config. So I'd like to propose
```
The sample size can be controlled by the config `xxx`
```

Reply:
@cloud-fan the sentence has been changed according to your suggestion (in both Spark & PySpark).